### PR TITLE
Adding sort to reorder webhookDescription based on webhookType

### PR DESCRIPTION
**Pull request by sujil02**
Adding sort to reorder webhookDescription based on webhookType
https://issues.redhat.com/browse/OSDK-1916

#5022 
Signed-off-by: sujil02 <sujil.shah@gmail.com>

**Description of the change:**

**Motivation for the change:**

**Checklist**

If the pull request includes user-facing changes, extra documentation is required:
- [ ] Add a new changelog fragment in `changelog/fragments` (see [`changelog/fragments/00-template.yaml`](https://github.com/operator-framework/operator-sdk/tree/master/changelog/fragments/00-template.yaml))
- [ ] Add or update relevant sections of the docs website in [`website/content/en/docs`](https://github.com/operator-framework/operator-sdk/tree/master/website/content/en/docs)

### CODE REVIEW

**Review by camilamacedo86 — CHANGES_REQUESTED** [review]
Hi @sujil02, 

For your contribution to be accepted you need to:
- Add [changelog entry](https://github.com/operator-framework/operator-sdk/blob/master/changelog/fragments/00-template.yaml) to generate the release notes. 
- Run [make generate](https://github.com/operator-framework/operator-sdk/blob/master/Makefile#L39) that will re-gen the docs and the samples in the testdata.

**Comment by sujil02** [comment]
> Hi @sujil02,
> 
> For we are able to merge your contribution you need to:
> 
> * Add [changelog entry](https://github.com/operator-framework/operator-sdk/blob/master/changelog/fragments/00-template.yaml) to generate the release notes.
> * Run [make generate](https://github.com/operator-framework/operator-sdk/blob/master/Makefile#L39) that will re-gen the docs and the samples in the testdata.

Done.

File: changelog/fragments/fix_order_of_generated_webhooks.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+entries:
+  - description: >
+      Adding sort to reorder webhookDescription based on webhookType
+      Ensuring generated webhook order is stable.
```
Comment (varshaprasad96) [inline changelog/fragments/fix_order_of_generated_webhooks.yaml:4]:
```suggestion
      Preserve the order of `webhookdefinitions` while generating bundles, by sorting `webhookDescriptions` in CSV generator.
```

**Review by varshaprasad96 — COMMENTED** [review]
just a small nit, looks good.
/lgtm

File: changelog/fragments/fix_order_of_generated_webhooks.yaml (hunk 1 above)
Comment (sujil02) [reply changelog/fragments/fix_order_of_generated_webhooks.yaml]: Fixed. Thank you.

**Review by rashmigottipati — APPROVED** [review]
/lgtm

File: changelog/fragments/fix_order_of_generated_webhooks.yaml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+entries:
+  - description: >
+      Preserve the order of `webhookdefinitions` while generating bundles, 
+      by sorting `webhookDescriptions` in CSV generator.
+
+    kind: bugfix
+    breaking: false
```
Comment (camilamacedo86) [inline changelog/fragments/fix_order_of_generated_webhooks.yaml:7]: nit space

**Review by camilamacedo86 — APPROVED** [review]
/lgtm

**Review by camilamacedo86 — COMMENTED** [review]
/lgtm

File: changelog/fragments/fix_order_of_generated_webhooks.yaml (hunk 2 above)
Comment (sujil02) [reply changelog/fragments/fix_order_of_generated_webhooks.yaml]: Fixed.

**Pull request merged**